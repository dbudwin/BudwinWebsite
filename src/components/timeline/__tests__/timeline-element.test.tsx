import React from 'react'
import { FaCode } from 'react-icons/fa'
import TimelineElement from '../timeline-element'
import renderer from 'react-test-renderer'

it('renders a single element in the timeline', () => {
  const timelineElement = renderer
    .create(
      <TimelineElement
        header='Foo'
        subtitle='Bar'
        date='2020'
        icon={<FaCode />}
      >
        Hello World
      </TimelineElement>
    )
    .toJSON()

  expect(timelineElement).toMatchSnapshot()
})
