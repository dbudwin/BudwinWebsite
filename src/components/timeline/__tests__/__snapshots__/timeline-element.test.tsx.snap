// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders a single element in the timeline 1`] = `
<div
  className="vertical-timeline-element"
  id=""
  style={null}
>
  <span
    className="vertical-timeline-custom-icon vertical-timeline-element-icon is-hidden"
    onClick={null}
    style={null}
  >
    <i
      className="fa"
    />
  </span>
  <div
    className="vertical-timeline-element-content is-hidden"
    onClick={null}
    style={null}
  >
    <div
      className="vertical-timeline-element-content-arrow"
      style={null}
    />
    <div
      className="box"
    >
      <h1
        className="title is-4"
      >
        Foo
      </h1>
      <h1
        className="subtitle is-6"
      >
        Bar
      </h1>
      <div
        className="content"
      >
        Hello World
      </div>
    </div>
    <span
      className="is-size-5 vertical-timeline-element-date"
    >
      2020
    </span>
  </div>
</div>
`;
